src/Simplex/Chat.hs

```diff
@@ -1633,6 +1632,7 @@ processAgentMessage (Just user@User {userId, profile}) corrId agentConnId agentM
         MSG msgMeta _msgFlags msgBody -> do
           cmdId <- createAckCmd conn
           msg@RcvMessage {chatMsgEvent = ACME _ event} <- saveRcvMSG conn (ConnectionId connId) msgMeta msgBody cmdId
+          asks chatLock >>= \l -> void . atomically . swapTMVar l $ "directMessage " <> B.unpack (strEncode $ toCMEventTag event)
```

Comment:
maybe do same in processGroupMessage

Reply:
also maybe do it first before `cmdId <- createAckCmd conn`?

Reply:
added to group; there is no `event` before createAckCmd